include/vapor/VolumeParams.h

```diff
@@ -54,6 +55,17 @@ class PARAMS_API VolumeParams : public RenderParams {
     //
     virtual size_t GetRenderDim() const override { return (3); }
 
+    //! \copydoc RenderParams::GetActualColorMapVariableName()
+    virtual string GetActualColorMapVariableName() const override
+    {
+        if (STLUtils::Contains(STLUtils::ToLower(GetAlgorithm()), "ospray"))
```

Comment:
Can we do something more robust here? Checking for a hard-coded string name is pretty ugly

Reply:
Done.